dart-impl/mpi/include/dash/dart/mpi/dart_globmem_priv.h

```diff
@@ -6,6 +6,7 @@
 #define DART_GPTR_COPY(gptr_, gptrt_)                       \
   do {                                                      \
     gptr_.unitid = gptrt_.unitid;                           \
+    gptr_.flags  = gptrt_.flags;                            \
```

Comment:
Why do we need this macro anyway? `gptr` can be directly assigned so there is no need to have a dedicated copy macro (which is not used anyway afair).